src/Components/Admin/Rooms/roomModal.js

```diff
@@ -341,7 +344,9 @@ export default function RoomModal({
             });
         }
 
-        resetFields();
+        if (!isEnvBar) {
+            resetFields();
+        }
```

Comment:
Fixed the bug. The issue was that on home page when you open share and publish, the room is being set which redefines the fields and resets the fields on each modal close. However once in the environment editor, the room isn't being set on each open so we cannot reset. This change makes sure that doesn't happen.

Ready for retesting and approval now @drbalar